test-common/uk/gov/hmrc/economiccrimelevyregistration/EclTestData.scala

```diff
@@ -599,4 +603,34 @@ trait EclTestData {
         )
       )
     }
+
+  implicit val arbValidNonUkEstablishmentRegistration: Arbitrary[ValidNonUkEstablishmentRegistration] = Arbitrary {
+    for {
+      businessName           <- Arbitrary.arbitrary[String]
+      companyNumber          <- Arbitrary.arbitrary[String]
+      utrType                <- Arbitrary.arbitrary[UtrType]
+      ctUtr                  <- Arbitrary.arbitrary[String]
+      saUtr                  <- Arbitrary.arbitrary[String]
+      taxIdentifier          <- Arbitrary.arbitrary[String]
+      commonRegistrationData <- Arbitrary.arbitrary[CommonRegistrationData]
+    } yield ValidNonUkEstablishmentRegistration(
+      commonRegistrationData.registration.copy(
+        entityType = Some(Other),
+        incorporatedEntityJourneyData = None,
+        partnershipEntityJourneyData = None,
+        soleTraderEntityJourneyData = None,
+        optOtherEntityJourneyData = Some(
+          commonRegistrationData.registration.otherEntityJourneyData.copy(
+            entityType = Some(NonUKEstablishment),
+            businessName = Some(businessName),
+            companyRegistrationNumber = Some(companyNumber),
+            utrType = Some(utrType),
+            ctUtr = Some(ctUtr),
```

Comment:
This is valid non UK establishment reg. It can't have both `ctutr` and `sautr` in the same time. Generate one of these here and update arbitrary in test case to match usage that you need.

Reply:
Amended.